### PR TITLE
Features hw 24 26

### PR DESCRIPTION
**Pull request by CyberCuCuber**
HW 24-26
Description in Readme.md

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ authors = ["RusLAN Polikarpov <ruslan.polikarpov.2014@gmail.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.10"
```
Comment (ggramal) [inline pyproject.toml:9]: why change python version here?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: "3.4"
+
+services:
+  authn_service:
+    image: cybercucumber/authn_service:latest
+    ports:
+      - 5000:5000
```
Comment (ggramal) [inline docker-compose.yml:7]: do we need to expose 5000 port here

File: ansible/roles/clone-git-repo/tasks/main.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    path: "{{ destination }}"
+    state: directory
+    recurse: yes
+    mode: 0755
+
+- name: Clone repo from git  
+  git:
+    repo: "{{ git_repo_link }}"
+    dest: "{{ destination }}"
+    version: features-hw-24-26
```
Comment (ggramal) [inline ansible/roles/clone-git-repo/tasks/main.yaml:12]: lets use var here to specify what branch we want to deploy

Comment (CyberCuCuber) [reply ansible/roles/clone-git-repo/tasks/main.yaml]: > lets use var here to specify what branch we want to deploy

Ok, i made this

Comment (CyberCuCuber) [reply ansible/roles/clone-git-repo/tasks/main.yaml]: > lets use var here to specify what branch we want to deploy

I don't quite understand what you mean

File: docker-compose.yml (hunk 2 above)
Comment (CyberCuCuber) [reply docker-compose.yml:7]: > do we need to expose 5000 port here

I don't quite understand what you mean

Comment (CyberCuCuber) [reply docker-compose.yml]: > do we need to expose 5000 port here

Fixed

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+version: "3.4"
+
+services:
+  authn_service:
+    image: cybercucumber/authn_service:latest
+
+  nginx:
+    container_name: nginx
```
Comment (ggramal) [inline docker-compose.yml:8]: We need a way to access logs without connecting to nginx container. So probably we should use volumes here to mount host filesystem to nginx

Comment (CyberCuCuber) [reply docker-compose.yml:8]: > We need a way to access logs without connecting to nginx container. So probably we should use volumes here to mount host filesystem to nginx

I have added volume to nginx container. Logs locate /home/authn/repo/nginx-logs. File authn.json.log

**Review by ggramal — APPROVED** [review]
LGTM

**Pull request merged**